### ISSUE TITLE
Dailies on Wrong Days (Custom Clock was not altered)

### ISSUE THREAD

**Issue by patach** [HabitRPG/habitica#1359]
It seems for the last couple days, my dailies have been popping out and greying out on the wrong days.  It seems as if the dailies are a day behind.

I haven't touched the Custom Clock, and am wondering if there's a way I could reset or set my date appropriately.  

Thank you in advanced for any help.

**Comment by Shaners** [comment]
Have you been using Habit on multiple computers? Likely this is related to all the derby issues which will hopefully be resolved after the move to angular.

Also have you checked to make sure that the correct days are selected for your dailies? For example in the attached this daily is only required to be completed Monday and Tuesday:
![daily](https://f.cloud.github.com/assets/2374703/1018812/5adbcdce-0c32-11e3-94bb-1a5f42559aa7.png)

**Comment by patach** [comment]
I have used Habitrpg through the Android App as well as the computer.  Maybe that's the issue?  Is there a way to sync it up?

I have checked my daily selection.  Like right now, I have one that is on Mondays, Wednesday and Fridays, and I see those days selected like in the picture you attached... and yet the daily is still greyed out.  It wasn't greyed out on Thursday though.  It's almost acting like it's a day behind.

**Comment by Shaners** [comment]
See  #1057 
It's probably related to derby issues so there isn't much that can be done until the Angular rewrite is completed. My guess is that the client is 'holding onto' a daily when you check it so when you return the next day or 2 days it thinks you've already completed that daily. You aren't doing anything wrong but if you leave the app or browser tab open for a longer period of time you could try to hit refresh before clicking off your tasks and refresh again after you've completing marking off your dailies.

**Comment by lefnire** [comment]
This actually could be due to the recent cron change I made to take user timezone into consideration. @Shaners do things look correct on your account? @patach what's your uuid?

**Comment by Shaners** [comment]
@lefnire my account seems fine.

**Comment by patach** [comment]
@lefnire My userid is aab6da6d-f1f0-463c-9586-063311fd9242.

@Shaners I have been updating my check lists by refreshing the site before adding any changes to them for the last few weeks.

**Comment by msaglietto** [comment]
Hello guys .. it happening to me too .. today (Monday) is showing the ones from yesterday(Sunday)

**Comment by hilatron** [comment]
Same issue for me, as described by patach.  I checked that my dailies are set to the correct days and have not altered the custom clock.

User ID: 2b5bf6c9-db76-42c0-87bc-31080a93417e

**Comment by traceyourshadows** [comment]
I'm having the same issue and have been for the past month or two. I started using the Android app recently, but until then I only ever used Habit on one computer, one browser.

**Comment by lefnire** [comment]
Ah man... I've done all that I can to get custom timezone and daystart working, I've even written some extensive unit tests to make sure they work properly - all of them passing. It works for most people, and somehow seems to fall through the cracks for others. I dare say, after racking my brain on this for a very long time I'm gonna have to give this one up to the community to look into.

Here's the information on jumping in: https://github.com/lefnire/habitrpg/wiki/Contributing#habitrpg-shared . Here's the [shouldDo](https://github.com/HabitRPG/habitrpg-shared/blob/rewrite/script/helpers.coffee#L37) function, where this kicksoff

**Comment by devarops** [comment]
I'm experiencing the same problem. Dailies for today are grayed out (not "active").

**Comment by lefnire** [comment]
#1457 & #1057

**Comment by paglias** [comment]
Follow on #1057

**Comment by AnastasiaStabie** [comment]
I'm having the same issue. 
I'm noticing an issue in "Dailies" 

I have "1h Japanese Study" as one of my "Daily" quests and want to have it setup to be a goal only on mondays, wednesdays, and fridays. I set it up originally on my computer browser(Chrome Version 29.0.1547.66 m paired with Windows 7.) However, when I checked the same task on the mobile version the days (on/off) had been switched. It was now enabled for S.T,TH,Su. 

I have used this specific example, but all of my tasks set to certain days show as switched between the mobile and desktop versions of the site. I have included screenshots for you and can give additional info if needed.

User ID: f68ff456-0833-44df-b895-0d7e384c8fc4
